meeting-minutes/2024-12-18.md

```diff
@@ -0,0 +1,134 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 9 or more of the 17 voting members (including the officers).
+
+| First Name | Last Name  | Company                                                     | Role(s)                   | Present |
+|:-----------|:-----------|:------------------------------------------------------------|:--------------------------|:--------|
+| Adrian     | Diglio     | Microsoft                                                   | Voting Member             | ??      |
+| David      | Kemp       | National Security Agency                                    | Member                    | ??      |
+| Denny      | Page       | Individual                                                  | Voting Member             | ???     |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member             | ??      |
+| Feng       | Cao        | Oracle                                                      | Member                    | ???     |
+| Harin      | Sarda      | Cisco Systems                                               | Voting Member             | ???     |
+| Jautau     | White      | Microsoft                                                   | Voting Member             | ???     |
+| Jeremy     | Rickard    | Microsoft                                                   | Member                    | ??      |
+| Justin     | Murphy     | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair                  | ???     |
+| Kris       | Vandecruys | Cisco Systems                                               | Voting Member             | ???     |
+| Kunal      | Modasiya   | Qualys, Inc.                                                | Member                    | ??      |
+| Langley    | Rock       | Dell                                                        | Voting Member             | ???     |
```

Comment:
Indeterminable whether Langley still has voting rights.

Reply:
Need September minutes to determine Langley's status for December meeting.
If not present in September, would have lost voting rights in September.
Not present in October.
Present for November and December so if voting rights lost, he would regain them after today for January meeting.